ui/main.qml

```diff
@@ -295,50 +295,16 @@ StatusWindow {
         applicationWindow.requestActivate()
     }
 
-    SystemTrayIcon {
+    StatusTrayIcon {
         id: systemTray
-        visible: true
-        icon.source: {
-            if (production) {
-                if (Qt.platform.os == Constants.mac)
-                    return "imports/assets/png/status-logo-round-rect.png"
-                else
-                    return "imports/assets/png/status-logo-circle.png"
-            } else {
-                if (Qt.platform.os == Constants.mac)
-                    return "imports/assets/png/status-logo-dev-round-rect.png"
-                else
-                    return "imports/assets/png/status-logo-dev-circle.png"
-            }
-        }
-
-        onMessageClicked: {
-            if (Qt.platform.os === Constants.windows) {
-                applicationWindow.makeStatusAppActive()
-            }
-        }
-
-        menu: Menu {
-            MenuItem {
-                text: qsTr("Open Status")
-                onTriggered: {
-                    applicationWindow.makeStatusAppActive()
-                }
-            }
-
-            MenuSeparator {
-            }
-
-            MenuItem {
-                text: qsTr("Quit")
-                onTriggered: Qt.quit()
-            }
-        }
-
-        onActivated: {
-            if (reason !== SystemTrayIcon.Context) {
-                applicationWindow.makeStatusAppActive()
+        objectName: "systemTray"
+        onActivateApp: {
```

Comment:
Revert this part pls, it breaks activating the tray icon with middle mouse click which opens the window (instead of just opening the menu)

Reply:
Or, if there are problems with the (middle) click activation on mac, pls make the code conditional

Reply:
reverted. Looks like a qt bug. It will request activate with `SystemTrayIcon.Trigger` when the context menu is requested. The docs say it should request `SystemTrayIcon.Context`.

We can live with it and I'll leave it like this since it can change with future qt updates.